Fullstack-Development/README.md

```diff
@@ -0,0 +1,90 @@
+# Frontend Roadmap
+## Get strted with frontend
+- Check out the frontend development roadmap
+
+## Get started with backend
+- Check out the backend development roadmap
+
+## Learning Typescript
+### Learn Typescript
+- [Resource 1](https://www.codecademy.com/learn/learn-typescript)
+- [Resource 2](https://www.freecodecamp.org/news/learn-typescript-beginners-guide/)
+- [Resource 3](https://youtu.be/gp5H0Vw39yw)
+### Typescript documentation
```

Comment:
No need to add a seprate heading for documentations. just add it as `[Typescript docs](link)`